src/workspaces/ShareWorkspaceModal/Collaborator.tsx

```diff
@@ -99,7 +99,7 @@ export const AclInput: React.FC<AclInputProps> = (props: AclInputProps) => {
 
   return (
     <div style={{ display: 'flex', marginTop: '0.25rem', flexDirection: showRow ? 'row' : 'column' }}>
-      <div style={{ width: isAzureWorkspace ? 425 : 200 }}>
+      <div style={{ width: isAzureWorkspace && showRow ? 425 : 200 }}>
```

Comment:
Fixing a small UI bug in Azure that distorts the page layout!

**Azure**
<img width="719" alt="Screenshot 2025-01-22 at 2 30 39 PM" src="https://github.com/user-attachments/assets/ddfc21a2-1615-469e-b8ff-65dfa6f9c643" />

**GCP**
<img width="719" alt="Screenshot 2025-01-22 at 2 30 53 PM" src="https://github.com/user-attachments/assets/75351350-7060-4b92-a6eb-da46801bef17" />